spec/features/review_spec.rb

```diff
@@ -3,12 +3,16 @@
 
 describe "review card process" do
   let!(:user) { create(:user, email: "u@mail.com", password: "123") }
-  let!(:card) { create(:card, translated_text: "Тест", user: user) }
+  let!(:deck_1) { create(:deck, user: user) }
+  let!(:deck_2) { create(:deck, user: user) }
+  let!(:card_1) { create(:card, translated_text: "Тест", user: user, deck: deck_1) }
+  let!(:card_2) { create(:card, translated_text: "Тост", user: user, deck: deck_2) }
 
   before(:each) do
     login("u@mail.com", "123")
-    card.update_attributes(review_date: Time.now)
-    card.update_attributes(picture: File.open("#{Rails.root}/spec/support/ruby.png"))
+    card_1.update_attributes(review_date: Time.now)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.